storico_ordini.html

```diff
@@ -0,0 +1,139 @@
+		<meta name="keyword" content="Ordini, prodotti, dettagli" />
+		<meta name="language" content="italian it"/>
+		<meta name="author" content="Dindinelli Alessandro, Tommasin Alessandro, Frison Nicol&ograve;, Giardina Mirco"/>
+	</head>
+	
+	<body>
+		<div id="header">
+            <span id="logo"></span>
+            <h1 xml:lang="ja">Sushi Nakamura</h1>
+        </div>
+
+        <div id="breadcrumb">
+			<p>Ti trovi in: <span xml:lang="en">Home</span> &gt; &gt; Storico Ordini</p>
+        </div>
+
+		<div id="menu">
+            <ul>
+                <li xml:lang="en"><a href="home_utente.html">Home</a></li>
+                <li xml:lang="en"><a href="take_away.html">Take Away</a></li>
+                <li><a href="chi_siamo.html">Chi siamo</a></li>
+                <li><a href="recensioni.html">Recensioni</a></li>
+                <li><a href="prodotti.html">Prodotti</a></li>
+                <li><a href="contatti.html">Contatti</a></li>
+                <li><a href="gestione_profilo_utente.html">Gestione profilo</a></li>
+            </ul>
+            <ul>
+                <li><a href="login.html"><span xml:lang="en" lang="en">Login</span>/Registrazione</a></li>
+                <li><a href="carrello.html">Carrello</a></li>
+                <li><input type="button" name="logout" id="logout" value="Logout"/></li>
+            </ul>
+        </div>
+		
+		<!-- Inizio contenuto -->
+		<div id="carrello" class="content">
+		
+			<!-- Inizio lista_ordini -->
+			<table>
+				<thead>
+					<tr>
+						<th>ID ordine</th>
+						<th>Data ordine</th>
+						<th>Data consegna</th>
+						<th>Totale</th>
+						<th>Operazioni</th>
+					</tr>
+				</thead>
+				<tbody>
+					<tr>
+						<td>ID</td>
+						<td>data1</td>
+						<td>data2</td>
+						<td>Totale</td>
+						<td>
+							<input type="button" name="dettagli" value="dettagli" />
+						</td>
+					</tr>
+					<tr>
+						<td colspan="5">
+							<!-- Inizio dettagli ordine -->
+							<div>
+								<p>Nome Cognome</p>
+								<p>Via, n. civico, citta, cap</p>
+								<p>Pagamento</p>
+								<table>
+									<thead>
+										<tr>
+											<th>Nome prodotto</th>
+											<th>Categoria</th>
+											<th>Porzioni</th>
+										</tr>
+									</thead>
+									<tbody>
+										<tr>
+											<td><a href="">Nome Prodotto</a></td>
+											<td><a href="">Categoria</a></td>
+											<td>Porzioni</td>
+										</tr>
+										<tr>
+											<td><a href="">Nome Prodotto</a></td>
+											<td><a href="">Categoria</a></td>
+											<td>Porzioni</td>
+										</tr>
+										<tr>
+											<td><a href="">Nome Prodotto</a></td>
+											<td><a href="">Categoria</a></td>
+											<td>Porzioni</td>
+										</tr>
+										<tr>
+											<td><a href="">Nome Prodotto</a></td>
+											<td><a href="">Categoria</a></td>
+											<td>Porzioni</td>
```

Comment:
Mancano i link alle categorie, andranno fatte dopo o possiamo già aggiungerli usando degli Id nella pagina prodotti?
Io direi di togliere il link dal nome del prodotto in quanto abbiamo spiegato solo le categorie e non i singoli prodotti